### PR TITLE
Session spider

### PR DESCRIPTION
**Pull request by oryband**

### CODE REVIEW

**Comment by oryband** [comment]
@dvirsky @akariv WHOO HOO!

File: scraper/scraper/settings.py [hunk 1]
```diff
@@ -13,9 +13,8 @@
     'scraper.pipelines.ResolutionPipeline': 300,
 }
 
-CONCURRENT_REQUESTS = 15
-CONCURRENT_REQUESTS_PER_DOMAIN = 30
-DOWNLOAD_DELAY = 0.50 # ms
+CONCURRENT_REQUESTS = 30
```
Comment (dvirsky) [inline scraper/scraper/settings.py:16]: they can withstand this?!

**Comment by dvirsky** [comment]
nicely done!

File: scraper/scraper/settings.py (hunk 1 above)
Comment (oryband) [inline scraper/scraper/settings.py:16]: from the 10-minute i ran i did, it looks like they do. maybe they previously crashed because we made bad requests.

**Pull request merged**